lib/dry/system/components.rb

```diff
@@ -0,0 +1,6 @@
+require 'dry/system'
+
+Dry::System.register_provider(
+  :system_components,
```

Comment:
I wonder if we could just call this `:system`? Perhaps we could make that our standard name for any system-provided components.

Reply:
@solnic Would you be OK to rename this?